arangod/RestHandler/RestShutdownHandler.cpp

```diff
@@ -68,6 +70,16 @@ RestStatus RestShutdownHandler::execute() {
     }
   }
 
+  auto const& softShutdownFeature{server().getFeature<SoftShutdownFeature>()};
+  auto& softShutdownTracker{softShutdownFeature.softShutdownTracker()};
+
+  if (_request->requestType() == rest::RequestType::GET) {
```

Comment:
Here we are not checking if we are a coordinator, but soft shutdown is only available there.
So on non-coordinators, the GET API will work but probably not return the expected results (I haven't checked if this is the case).
Should we be consistent here and return an error if the GET API is called on non-coordinators?

Reply:
Yes, let's do this, will do.

Reply:
Done.